src/Tests/Encoder/StreamEncoderTest.php

```diff
@@ -12,6 +12,8 @@
  */
 class StreamEncoderTest extends KernelTestBase {
 
+  protected $strictConfigSchema = FALSE;
```

Comment:
We have an schema error for PUT and HEAD methods when running tests - methods that are not defined by Rest module. Setting this to TRUE just avoid the check in `Drupal\Core\Config\Testing\ConfigSchemaChecker::onConfigSave()`.